widget/shared/deepLink.js

```diff
@@ -0,0 +1,71 @@
+class Deeplink {
```

Comment:
Why double class? You can leave it here only on widget/shared and you will be able to import it also from cp side.